ndarray/src/main/java/org/tensorflow/ndarray/Shape.java

```diff
@@ -108,13 +110,14 @@ public long size() {
    * an unknown size, {@link Shape#UNKNOWN_SIZE} is returned.
    *
    * @param i the index of the dimension to get the size for. If this Shape has a known number of
-   *     dimensions, it must be &lt; {@link Shape#numDimensions()}. The index may be negative, in which
-   *     case the position is counted from the end of the shape. E.g.: {@code size(-1)} returns the
-   *     size of the last dimension, {@code size(-2)} the size of the second to last dimension etc.
+   *     dimensions, it must be &lt; {@link Shape#numDimensions()}. The index may be negative, in
+   *     which case the position is counted from the end of the shape. E.g.: {@code size(-1)}
+   *     returns the size of the last dimension, {@code size(-2)} the size of the second to last
+   *     dimension etc.
    * @return The size of the dimension with the given index if known, {@link Shape#UNKNOWN_SIZE}
    *     otherwise.
    */
-  public long size(int i) {
+  public long get(int i) {
```

Comment:
Can you please preserve the `size` endpoint as deprecated? I know we are still in alpha mode but still that would easily guide our users to migrate from one version to another without breaking their code at each release.